### PR TITLE
NotificationHub Sample Template

### PR DESCRIPTION
**Pull request by smithab**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@smithab__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: 101-notification-hubs/azuredeploy.json [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+        "notificationHubName": {
+            "type": "string",
+            "metadata": {
+                "description": "Name of the NotificationHub"
+            }
+        }
+    },
+    "variables": {
+        "location": "[resourceGroup().location]",
+        "nhVersion": "[providers('Microsoft.NotificationHubs', 'namespaces').apiVersions[0]]",
```
Comment (singhkay) [inline 101-notification-hubs/azuredeploy.json]: Neat trick! However is there a possibility that this template will break automatically if a breaking change is introduced to a future api version?

File: 101-notification-hubs/readme.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+# Create a Namespace and NotificationHub  using a template
+
+
```
Comment (singhkay) [inline 101-notification-hubs/readme.md]: Can you add some description to the readme?

**Comment by singhkay** [comment]
@smithab any update on this?

**Comment by smithab** [comment]
I'll work on this changes this week.

**Comment by singhkay** [comment]
@smithab Sounds good. If you're MSFT FTE, please send me a mail offline so I can take care of the CLA.

**Comment by singhkay** [comment]
@smithab Can you sign the CLA to enable us to review and merge this?

**Comment by singhkay** [comment]
Closing as there hasn't been an update on this in a while. Feel free to reopen with an update.

**Pull request closed without merging**